### PR TITLE
feat(ca): add identifiers confId, mid

### PR DESCRIPTION
**Pull request by chrisadubois**
# COMPLETES 
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-452762
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-462077
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-473895

## This pull request addresses

Add globalMeetingId and webexConferenceIdStr from either meetingInfo when a meeting object is available, OR from options passed in by consumers

## by making the following changes
updating the client metrics plugin
adding the params directly from the meetinginfo.response
update types

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

Manual tests (guest and login) meeting joins

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ const userAgent = `webex-js-sdk/test-webex-version client=Cantina; (os=${getOSNa
   getOSVersion().split('.')[0]
 })`;
 
-describe('internal-plugin-metrics', () => {
+describe.only('internal-plugin-metrics', () => {
```
Comment (torgeadelin) [inline packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts:20]: remove .only

File: packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts [hunk 2]
```diff
@@ -362,6 +362,8 @@ export default class MeetingInfoV2 {
             payload: {
               identifiers: {
                 meetingLookupUrl: response?.url,
+                webexConferenceIdStr: response?.body?.confID,
```
Comment (torgeadelin) [inline packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts:365]: use options instead of payload

**Review by torgeadelin — CHANGES_REQUESTED** [review]
few blocking comments, but looking good

File: packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts (hunk 1 above)
Comment (chrisadubois) [reply packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts:20]: nice catch ty

File: packages/@webex/plugin-meetings/src/meeting-info/index.ts [hunk 3]
```diff
@@ -96,6 +96,8 @@ export default class MeetingInfo {
             payload: {
               identifiers: {
                 meetingLookupUrl: info?.url,
+                webexConferenceIdStr: info?.body?.confID,
```
[the remark this answers is not in the document]
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/src/meeting-info/index.ts]: done

**Comment by chrisadubois** [comment]
all requested changes complete

**Review by torgeadelin — APPROVED** [review]

**Review by robstax — APPROVED** [review]
small typo in test description, but non-blocking

**Pull request merged**